### DISCUSSION TITLE
Node Ownership for specified User

### DISCUSSION THREAD

**Discussion by Oldbear200** [pelican-dev/panel#21] [Ideas]
Hey please add that admins can assign a node to a user so he can create / manage / delete services with just that node.
Would love that

**Comment by Doc94** [comment]
Sounds great maybe to allow more than one user manage servers in specifics nodes and not a full nodes... Not sure if that use case was mentioned in another place

**Comment by lancepioch** [comment]
Possibilities:

Scope server creation to: nodes.user_id or nodes.owner_id or node_user.access_id

Scope server viewing to: where('servers.user_id', auth()->user()->id)

**Comment by lancepioch** [comment]
We are absolutely working on this now!

**Reply by Oldbear200** [reply]
Thank you so much

**Reply by PsychoZander** [reply]
Do you have an issue number so we can follow? I can't see to find one that is relevant.

**Reply by rmartinoscar** [reply]
> Do you have an issue number so we can follow? I can't see to find one that is relevant.

Here you go
https://github.com/pelican-dev/panel/issues/613
